sed/core/logging.py

```diff
@@ -0,0 +1,67 @@
+"""
+This module provides a function to set up logging for the application. It configures
+both console and file logging handlers, allowing different log levels for each. The
+log files are stored in a user-specific log directory.
+
+"""
+import logging
+
+from sed.core.user_dirs import USER_LOG_PATH
+
+
+def setup_logging(name: str, verbose: bool = False, debug: bool = False) -> logging.Logger:
+    """
+    Configures and returns a logger with specified log levels for console and file handlers.
+
+    Args:
+        name (str): The name of the logger.
+        verbose (bool): If True, sets the console log level to INFO. Defaults to False.
+        debug (bool): If True, sets the console log level to DEBUG. Defaults to False.
```

Comment:
Why not just use one parameter "log_level"?

Reply:
I wanted to do that but since SedProcessor currently has a bool flag for verbose I tried to keep the compatibility. We can definitely change that.

Reply:
Okay, we can change this then once we integrate this logging into the other modules